NEW_README.md

```diff
@@ -75,3 +76,17 @@ From root of the gatling-puppet-load-test source dir on the Gatling driver (metr
     * scp them locally and you can view them in your browser.
 * atop files containing cpu, mem, disk and network usage overall and broken down per process are available to view on the mom: atop -r /var/log/atop/atop_\<date>
     * See the atop man file for interactive commands
+    
+### Aborting a test run
+The easiest way to immediately kill a test run is to kill the corresponding Java process on the metrics node. Find the process id with top and then use `kill <PID>`.
+
+### Generating reports for aborted runs
+If a Gatling scenario is aborted the reports may not be generated. In this case you can run Gatling in reports-only mode to generate the reports for a previous run. 
+You'll need to specify the name of the folder on the metrics node containing the simulation.log file for the run. 
+Look in `~/gatling-puppet-load-test/simulation-runner/results` for a folder that starts with 'PerfTestLarge-' followed by the run id and verify that the folder contains a valid simulation.log file.
```

Comment:
you say "valid" simulation log.  Is there something specific that needs to be in the file?  Or just anything at all makes it good?

Reply:
Mainly it has to contain data / can't be empty. During my initial testing I had empty log files which caused errors. Once I updated `gatling.data.file.bufferSize` to ensure that data was written during a short test run I had no further issues generating reports from aborted runs. It may be possible that if Gatling is killed while it is writing to the file that it could contain data but still be invalid, but I wasn't able to reproduce this.